docs/croissant_rai.ttl

```diff
@@ -0,0 +1,127 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix croissant: <http://mlcommons.org/croissant/> .
+@prefix rai: <http://mlcommons.org/croissant/RAI/> .
+
+# Dataset level properties
+
+rai:dataCollection a rdf:Property ;
+rdfs:label "dataCollection" ;
+rdfs:comment "Description of the data collection process." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text .
+
+rai:dataCollectionType a rdf:Property ;
+rdfs:label "dataCollectionType" ;
+rdfs:comment "Define the data collection type. Recommended values: Surveys, Secondary Data analysis, Physical data collection, Direct measurement, Document analysis, Manual Human Curator, Software Collection, Experiments, Web Scraping, Web API, Focus groups, Self-reporting, Customer feedback data, User-generated content data, Passive Data Collection, Others." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text, schema:ItemList .
+
+rai:dataCollectionMissingData a rdf:Property ;
+rdfs:label "dataCollectionMissingData" ;
+rdfs:comment "Description of missing data in structured/unstructured form." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text, schema:ItemList .
+
+rai:dataCollectionRawData a rdf:Property ;
+rdfs:label "dataCollectionRawData" ;
+rdfs:comment "Description of the raw data i.e. source of the data." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:Text, schema:ItemList .
+
+rai:dataCollectionTimeframe a rdf:Property ;
+rdfs:label "dataCollectionTimeframe" ;
+rdfs:comment "Timeframe in terms of start and end date of the collection process." ;
+schema:domainIncludes schema:Dataset ;
+schema:rangeIncludes schema:DateTime .
```

Comment:
This should specify how the time interval should be represented, e.g., following https://en.wikipedia.org/wiki/ISO_8601#Time_intervals. 

https://schema.org/temporalCoverage might be a good example to follow.

Reply:
Thanks. 
 We can add `schema:supersededBy schema:temporalCoverage ;` for defining time frame

Reply:
thanks for the comments, is it best to specify using rdfs:comment that the time interval should be indicated in ISO 8601 interval format?